### ISSUE TITLE
Question about using Deuterium over Tritium?

### ISSUE THREAD

**Issue by btedman** [fusion-energy/inertial_fusion_openmc_dagmc_paramak_example#13]
My project uses a spherical IEC reactor with a Deuterium fuel. Is there a way to replace the existing code so as to instantiate this?

**Comment by shimwell** [comment]
Current the example has a 14MeV source
https://github.com/fusion-energy/inertial_fusion_openmc_dagmc_paramak_example/blob/6d710616634f2430820a60363f54a26e21a80657/2_run_openmc_dagmc_simulation.py#L67
You can replace with with a 2.5MeV source

```python
my_source.energy = openmc.stats.Discrete([2.5e6], [1]) 
```

Or you can do something better with the openmc.stats.muir distribution
```python
my_source.energy = openmc.stats.muir(e0=2500000.0, m_rat=4.0, kt=20000.0)
```
I can offer more complex mixed sources if that helps

**Comment by btedman** [comment]
When you mention the source, are you referring to the electric potential generated, modelled as a point source, or discrete ions? Is this how you recommend implementing Deuterium over Tritium?

**Comment by btedman** [comment]
Essentially, my team has a spherical IEC device, and we use a deuterium fuel. I am curious if there are ways to modify the existing code to reflect our specific reactor shape, fuel source, and operating values (i.e. voltage, pressure, temp)? We need to develop a simulation model and submit results to our national board in order to gain experimental approval.

**Comment by btedman** [comment]
And we are specifically looking to model the Bremsstrahlung radiation/ X-Ray Emission?

**Comment by shimwell** [comment]
> When you mention the source, are you referring to the electric potential generated, modelled as a point source, or discrete ions? Is this how you recommend implementing Deuterium over Tritium?

openmc can offer photon or neutrons for the source term. The code is a monte carlo particle transport code and doesn't know about electric potential. To change the code to model neutrons from DD instead of neutrons from DT then you can simple change the neutron energy from 14MeV to 2.5MeV

**Comment by shimwell** [comment]
> Essentially, my team has a spherical IEC device, and we use a deuterium fuel. I am curious if there are ways to modify the existing code to reflect our specific reactor shape, fuel source, and operating values (i.e. voltage, pressure, temp)? We need to develop a simulation model and submit results to our national board in order to gain experimental approval.

Yes OpenMC and DAGMC can support different geometry types. Perhaps have a look at the documentation
https://docs.openmc.org/en/stable/usersguide/geometry.html

**Comment by shimwell** [comment]
> And we are specifically looking to model the Bremsstrahlung radiation/ X-Ray Emission?

Yes openmc can simulate photons. change the ```my_source.particle=='photon'``` as it defaults to neutron
